docs/api/browser-window.md

```diff
@@ -213,16 +213,13 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
     * `followWindow` - The backdrop should automatically appear active when the window is active, and inactive when it is not. This is the default.
     * `active` - The backdrop should always appear active.
     * `inactive` - The backdrop should always appear inactive.
-  * `titleBarStyle` String (optional) - The style of window title bar.
+  * `titleBarStyle` String (optional) - The style of window title bar on macOS or Windows.
```

Comment:
```suggestion
  * `titleBarStyle` String (optional) _macOS_ _Windows_ - The style of window title bar.
```

Putting the OS limits here is treated specially by our docs parsers, see e.g. https://www.electronjs.org/docs/api/dialog#dialogshowopendialogsyncbrowserwindow-options